### PR TITLE
Update delve-for-office-365-admins.md

### PR DESCRIPTION
**Pull request by Revathi-b**
The document has been updated with the recent changes.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/delve-for-office-365-admins.md [hunk 1]
```diff
@@ -25,60 +25,46 @@ description: "The more you and your colleagues use Microsoft 365 to work togethe
...
   
-Regardless of which Microsoft 365 or Office 365 subscriptions you have, you need to activate the SharePoint service and assign users a SharePoint license before they can start using Delve. You also have to set up Exchange Online if you want attachments to show up on users' Home pages in Delve. If you set up Skype for Business Online, users can start Skype for Business Online conversations directly from Delve. And if you set up Yammer Enterprise, users can engage in Yammer conversations about documents directly from Delve.
-  
-Delve is designed to work with the current or immediately previous version of Internet Explorer or Firefox, or the latest version of Chrome, or Safari.
+Regardless of which Microsoft 365 or Office 365 subscriptions you have, you need to activate the SharePoint service and assign users a SharePoint license before they can start using Delve. You also have to set up Exchange Online if you want attachments to show up on users' Home pages in Delve. If you set up Microsoft Teams, users can start Microsoft Teams conversations directly from Delve.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/delve-for-office-365-admins.md:37]:
```suggestion
Regardless of which Microsoft 365 or Office 365 subscriptions you have, you need to activate the SharePoint service and assign users a SharePoint license before they can start using Delve. You also have to set up Exchange Online if you want attachments to show up on users' Home pages in Delve. If you set up Microsoft Teams, users can start Microsoft Teams conversations directly from Delve.
```

File: SharePoint/SharePointOnline/delve-for-office-365-admins.md [hunk 2]
```diff
@@ -25,60 +25,46 @@ description: "The more you and your colleagues use Microsoft 365 to work togethe
...
-  
-What users see in Delve is influenced by which release program you've chosen for your organization. For an overview of the main differences, see [My Delve looks different from what you describe](https://support.office.com/article/8fe977a0-cc37-4845-a75d-96994dffca5d). 
-  
-## Control access to Delve and related features
+## Control access to Delve
 <a name="BKMK_DelveOnOff"> </a>
 
-You control access to Delve from the SharePoint admin center. By default, users in your organization have access to Delve. They also have access to certain features in other apps that are powered by the Office Graph, such as the "Suggested" list on the SharePoint start page. When you enable or disable Delve, these other features are also affected. 
+You control access to Delve from the SharePoint admin center. By default, users in your organization have access to Delve. Users can limit their Delve experience to only show profile information by turning off **Show documents in Delve** under **Settings** > **Feature settings** in Delve. As an admin, you can also remove documents from the Delve through the item insights privacy settings.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/delve-for-office-365-admins.md:47]:
```suggestion
You control access to Delve from the SharePoint admin center. By default, users in your organization have access to Delve. Users can limit their Delve experience to only show profile information by turning off **Show documents in Delve** under **Settings** > **Feature settings** in Delve. As an admin, you can also remove documents from Delve through the item insights privacy settings.
```

File: SharePoint/SharePointOnline/delve-for-office-365-admins.md [hunk 3]
```diff
@@ -25,60 +25,46 @@ description: "The more you and your colleagues use Microsoft 365 to work togethe
...
 
-If you disable Delve, users will no longer see:
-- The Delve app tile in the app launcher.
-- Suggested sites on the SharePoint start page.
-- The Recommended feed in the Outlook mobile app.
-- Recommended documents on the Microsoft Office Home page.
-- Documents on other users' profile cards and pages.
+For more info on item insights privacy settings, see [Customizing item insights privacy in Microsoft Graph (preview)](https://docs.microsoft.com/en-us/graph/insights-customize-item-insights-privacy)
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/delve-for-office-365-admins.md:49]:
```suggestion
For more info on item insights privacy settings, see [Customizing item insights privacy in Microsoft Graph (preview)](/graph/insights-customize-item-insights-privacy).
```

File: SharePoint/SharePointOnline/delve-for-office-365-admins.md [hunk 4]
```diff
@@ -25,60 +25,46 @@ description: "The more you and your colleagues use Microsoft 365 to work togethe
...
 
- For more info, see [What is the effect of enabling or disabling Delve and related features?](delve-for-office-365-admins.md#BKMK_EffectOfficegraphOnOff). 
+For more info on disabling Delve, see [What is the effect of enabling or disabling Delve](https://docs.microsoft.com/en-us/sharepoint/delve-for-office-365-admins%22%20/l%20%22BKMK_EffectOfficegraphOnOff).
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/delve-for-office-365-admins.md:51]:
```suggestion
For more info on disabling Delve, see [What is the effect of enabling or disabling Delve](/sharepoint/delve-for-office-365-admins%22%20/l%20%22BKMK_EffectOfficegraphOnOff).
```

File: SharePoint/SharePointOnline/delve-for-office-365-admins.md [hunk 5]
```diff
@@ -25,60 +25,46 @@ description: "The more you and your colleagues use Microsoft 365 to work togethe
...
   
-1. Go to the [Settings page of the new SharePoint admin center](https://admin.microsoft.com/sharepoint?page=settings&modern=true), and sign in with an account that has [admin permissions](./sharepoint-admin-role.md) for your organization.
+1. Go to the [Settings page of the new SharePoint admin center](https://admin.microsoft.com/sharepoint?page=settings&modern=true), and sign in with an account that has [admin permissions](https://docs.microsoft.com/en-us/sharepoint/sharepoint-admin-role) for your organization.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/delve-for-office-365-admins.md:53]:
```suggestion
1. Go to the [Settings page of the new SharePoint admin center](https://admin.microsoft.com/sharepoint?page=settings&modern=true), and sign in with an account that has [admin permissions](/sharepoint/sharepoint-admin-role) for your organization.
```

File: SharePoint/SharePointOnline/delve-for-office-365-admins.md [hunk 6]
```diff
@@ -25,60 +25,46 @@ description: "The more you and your colleagues use Microsoft 365 to work togethe
...
 
->[!NOTE]
->If you have Office 365 Germany, [sign in to the Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=848041), then browse to the SharePoint admin center, and open the **Settings** page. <br>If you have Office 365 operated by 21Vianet (China), [sign in to the Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=850627), then browse to the SharePoint admin center, and open the **Settings** page.
+    >[!NOTE]
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/delve-for-office-365-admins.md:55]:
```suggestion
    > [!NOTE]
```

File: SharePoint/SharePointOnline/delve-for-office-365-admins.md [hunk 7]
```diff
@@ -25,60 +25,46 @@ description: "The more you and your colleagues use Microsoft 365 to work togethe
...
+    >If you have Office 365 Germany, [sign in to the Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=848041), then browse to the SharePoint admin center,        and open the **Settings** page. <br>If you have Office 365 operated by 21Vianet (China), [sign in to the Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?          linkid=850627), then browse to the SharePoint admin center, and open the **Settings** page.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/delve-for-office-365-admins.md:56]:
```suggestion
    > If you have Office 365 Germany, [sign in to the Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=848041), then browse to the SharePoint admin center and open the **Settings** page. <br>If you have Office 365 operated by 21Vianet (China), [sign in to the Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=850627), then browse to the SharePoint admin center and open the **Settings** page.
```

File: SharePoint/SharePointOnline/delve-for-office-365-admins.md [hunk 8]
```diff
@@ -25,60 +25,46 @@ description: "The more you and your colleagues use Microsoft 365 to work togethe
...
-3.  Under **Delve**, select one of the following:  
+3.  Under **Delve**, select one of the following:
     
-  - **Enable Delve**
+    - **Enable Delve**
     
-  - **Disable Delve**
+    - **Disable Delve**
+  
+    >[!NOTE]
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/delve-for-office-365-admins.md:66]:
```suggestion
    > [!NOTE]
```

File: SharePoint/SharePointOnline/delve-for-office-365-admins.md [hunk 9]
```diff
@@ -25,60 +25,46 @@ description: "The more you and your colleagues use Microsoft 365 to work togethe
...
+    >Previously, when turning Delve on or off in the SharePoint admin center, it also had an impact on a few other insights-driven experiences in Microsoft 365. Starting July      1, 2021, the Delve setting will be decoupled from the back end and will only impact Delve. Use the item insights privacy settings to control item insights in Microsoft 365.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/delve-for-office-365-admins.md:67]:
```suggestion
    > Previously, when turning Delve on or off in the SharePoint admin center, it also had an impact on a few other insights-driven experiences in Microsoft 365. Starting July 1, 2021, the Delve setting will be decoupled from the back end and will only impact Delve. Use the item insights privacy settings to control item insights in Microsoft 365.
```

File: SharePoint/SharePointOnline/delve-for-office-365-admins.md [hunk 10]
```diff
@@ -134,12 +120,12 @@ There are a few things you should check if one or more users in your organizatio
   
 1. Go to the [Settings page of the new SharePoint admin center](https://admin.microsoft.com/sharepoint?page=settings&modern=true), and sign in with an account that has [admin permissions](./sharepoint-admin-role.md) for your organization.
 
->[!NOTE]
->If you have Office 365 Germany, [sign in to the Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=848041), then browse to the SharePoint admin center and open the Settings page. <br>If you have Office 365 operated by 21Vianet (China), [sign in to the Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=850627), then browse to the SharePoint admin center and open the Settings page.
+    >[!NOTE]
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/delve-for-office-365-admins.md:123]:
```suggestion
    > [!NOTE]
```

File: SharePoint/SharePointOnline/delve-for-office-365-admins.md [hunk 11]
```diff
@@ -134,12 +120,12 @@ There are a few things you should check if one or more users in your organizatio
...
+    >If you have Office 365 Germany, [sign in to the Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=848041), then browse to the SharePoint admin center and open the Settings page. <br>If you have Office 365 operated by 21Vianet (China), [sign in to the Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=850627), then browse to the SharePoint admin center and open the Settings page.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/delve-for-office-365-admins.md:124]:
```suggestion
    > If you have Office 365 Germany, [sign in to the Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=848041), then browse to the SharePoint admin center and open the Settings page. If you have Office 365 operated by 21Vianet (China), [sign in to the Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=850627), then browse to the SharePoint admin center and open the Settings page.
```

File: SharePoint/SharePointOnline/delve-for-office-365-admins.md [hunk 12]
```diff
@@ -150,45 +136,45 @@ There are a few things you should check if one or more users in your organizatio
...
 3. Verify that you have one of the following subscriptions:
     
-  - Office 365 Enterprise (E1, E3, or E5)
+    - Office 365 Enterprise (E1, E3, or E5)
     
-  - Office 365 Education
+    - Office 365 Education
     
-  - Office 365 Government (E1, E3 or E5)
+    - Office 365 Government (E1, E3 or E5)
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/delve-for-office-365-admins.md:149]:
```suggestion
    - Office 365 Government (E1, E3, or E5)
```

File: SharePoint/SharePointOnline/delve-for-office-365-admins.md [hunk 13]
```diff
@@ -216,12 +202,12 @@ The user pictures in Delve are from the SharePoint user profiles. If there's no
   
 - Make sure that users upload their user profile picture to SharePoint. For more info, point users to [View and update your profile in Delve](https://support.office.com/article/4e84343b-eedf-45a1-aeb9-8627ccca14ba). 
     
-### Users see documents from other users who have turned off Documents in Delve
+### Users see documents from other users who have turned off documents in Delve
 <a name="BKMK_DelveOff"> </a>
 
- Users can turn off **Documents** in Delve, which means that they won't see any documents in Delve, and their **activities** and **relationships** won't be included in the Office Graph.
+ Users can turn off documents in Delve, which means that they won't see any documents in Delve, and their **activities** and **relationships** won't be included in the Microsoft  Graph.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/delve-for-office-365-admins.md:208]:
```suggestion
 Users can turn off documents in Delve, which means that they won't see any documents in Delve, and their **activities** and **relationships** won't be included in Microsoft  Graph.
```

File: SharePoint/SharePointOnline/delve-for-office-365-admins.md [hunk 14]
```diff
@@ -216,12 +202,12 @@ The user pictures in Delve are from the SharePoint user profiles. If there's no
...
   
-However, if other users still have  *access*  to documents from a user who has turned off **Documents**, they can still see those documents in Delve, just as they can search for them in SharePoint.
+However, if other users still have *access* to documents from a user who has turned off documents, they can still see those documents other places in Delve, just as they can search for them in SharePoint.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/delve-for-office-365-admins.md:210]:
```suggestion
However, if other users still have *access* to documents from a user who has turned off documents, they can still see those documents in Delve, just as they can search for them in SharePoint.
```

File: SharePoint/SharePointOnline/delve-for-office-365-admins.md [hunk 15]
```diff
@@ -256,54 +242,29 @@ Delve doesn't change any permissions and users will only see what they already h
...
-- **Trending Insight API** (trending documents for all users).
-- **Used Insight API** (what documents other users have viewed or modified).
-
-**NOTE:** Disabling Delve does not disable **Microsoft Graph**. Office Graph and Microsoft Graph are different concepts despite their similar names. Microsoft Graph is the API endpoint that surfaces all Microsoft 365 data (such as Azure AD, Outlook/Exchange, Excel, OneDrive, SharePoint), while Office Graph is mainly a code name for the collective set of services and insights generated on top of the infrastructure developed by the FAST Office Graph team. For more info about Microsoft Graph, see [Overview of Microsoft Graph](/graph/overview).
-
+> [!NOTE]
+> Disabling Delve does no longer disable *Microsoft Graph* or insights services generated on top of it. This level of control is now available from the item insights privacy settings. For more info about the item insights settings, see [Customizing item insights privacy in Microsoft Graph](https://docs.microsoft.com/en-us/graph/insights-customize-item-insights-privacy). For more info about Microsoft Graph, see [Overview of Microsoft Graph](https://docs.microsoft.com/en-us/graph/overview).
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/delve-for-office-365-admins.md:267]:
```suggestion
> Disabling Delve does no longer disable *Microsoft Graph* or insights services generated on top of it. This level of control is now available from the item insights privacy settings. For more info about the item insights settings, see [Customizing item insights privacy in Microsoft Graph](/graph/insights-customize-item-insights-privacy). For more info about Microsoft Graph, see [Overview of Microsoft Graph](/graph/overview).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Revathi-b, kindly commit my suggested changes. Please note, links to other topics in docs need to be relative links, as shown in my suggested changes. Many thanks.

File: SharePoint/SharePointOnline/delve-for-office-365-admins.md (hunk 14 above)
Comment (Revathi-b) [reply SharePoint/SharePointOnline/delve-for-office-365-admins.md]: I removed **other places** word as suggested, but there is no specific action for documents, so we did not bold the word documents.

**Comment by Revathi-b** [comment]
> Copy review - @Revathi-b, kindly commit my suggested changes. Please note, links to other topics in docs need to be relative links, as shown in my suggested changes. Many thanks.

@JohanFreelancer9, I have incorporated all of your suggestions, and I am requesting your approval to publish the document.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by Revathi-b** [comment]
@garycentric We have updated the document with the most recent changes, and JohanFreelancer9 has approved the content. I am requesting you to merge and publish the document. Thanks

**Comment by scanum** [comment]
@GreteLittmann PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**